### PR TITLE
[Docs] Added -master-addresses parameter to yb-admin commands under explore/PITR DB-5234

### PR DESCRIPTION
**Pull request by vkulichenko**

### CODE REVIEW

File: docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ Create and populate a table, get a timestamp to which you'll restore, and then w
 1. Create a snapshot schedule for the new `pitr` keyspace from a shell prompt. In the following example, the schedule is one snapshot every minute, and each snapshot is retained for ten minutes:
 
     ```sh
-    ./bin/yb-admin create_snapshot_schedule 1 10 ycql.pitr
+    ./bin/yb-admin -master_addresses <ip1:7100,ip2:7100,ip3:7100> create_snapshot_schedule 1 10 ycql.pitr
```
Comment (polarweasel) [inline docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md:103]: The angle brackets are definitely going to make all of these commands fail! Please test out commands before you add them to docs :)

```suggestion
    ./bin/yb-admin -master_addresses ip1:7100,ip2:7100,ip3:7100 create_snapshot_schedule 1 10 ycql.pitr
```

File: docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ Create and populate a table, get a timestamp to which you'll restore, and then w
 1. Verify that a snapshot has happened:
 
     ```sh
-    ./bin/yb-admin list_snapshot_schedules
+    ./bin/yb-admin -master_addresses ip1:7100,ip2:7100,ip3:7100 \ list_snapshot_schedules
```
Comment (ddorian) [inline docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md:107]:
```suggestion
    ./bin/yb-admin -master_addresses <ip1:7100,ip2:7100,ip3:7100> \ list_snapshot_schedules
```

Comment (ddorian) [reply docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md:107]: In some places it's with `<ip1:7100,ip2:7100,ip3:7100>`, in some places it's `ip1:7100,ip2:7100,ip3:7100`. I'd like it to be consistent, either version works but I'd prefer with `<>`

**Review by ddorian — APPROVED** [review]
LGTM except small issue

**Pull request merged**